shuup/admin/modules/service_providers/forms.py

```diff
@@ -22,3 +24,35 @@ class CustomPaymentProcessorForm(ShuupAdminForm):
     class Meta:
         model = CustomPaymentProcessor
         exclude = ("identifier", )
+
+
+class ServiceWizardFormMixin(object):
+    def save(self):
+        provider = super(ServiceWizardFormMixin, self).save()
+        if Shop.objects.count() == 1 and TaxClass.objects.count() == 1 and not self.service_exists(provider):
```

Comment:
Ok here you should probably include creating default class before creating service providers. Otherwise the services is never created.